sw-emulator/app/src/main.rs

```diff
@@ -12,12 +12,13 @@ Abstract:
 
```

Comment:
Can you change this commit title from "[fix] Emulator should use external mailbox bus" to "[fix] sw-emulator app should use external mailbox bus"

Reply:
This is already in main branch : https://github.com/chipsalliance/caliptra-sw/blob/main/sw-emulator/app/src/main.rs#L385

Reply:
Removed the commit under discussion.